### ISSUE TITLE
Example JIT code error

### ISSUE THREAD

**Issue by MaxDesiatov** [harlanhaskins/LLVMSwift#122]
On macOS 10.12.6 with Xcode 9.2 (9C40b) after installing llvm with `brew install llvm` and getting pkg-config instructions sorted out I try this example code:

```swift
import LLVM

let module = Module(name: "main")

let builder = IRBuilder(module: module)

let main = builder.addFunction(
             "main",
             type: FunctionType(argTypes: [],
             returnType: VoidType())
           )
let entry = main.appendBasicBlock(named: "entry")
builder.positionAtEnd(of: entry)

let function = builder.addFunction(
                 "calculateFibs",
                 type: FunctionType(argTypes: [IntType.int1],
                 returnType: FloatType.double)
               )
let entryBB = function.appendBasicBlock(named: "entry")
builder.positionAtEnd(of: entryBB)

// allocate space for a local value
let local = builder.buildAlloca(type: FloatType.double, name: "local")

// Compare to the condition
let test = builder.buildICmp(function.parameters[0], IntType.int1.zero(), .equal)

// Create basic blocks for "then", "else", and "merge"
let thenBB = function.appendBasicBlock(named: "then")
let elseBB = function.appendBasicBlock(named: "else")
let mergeBB = function.appendBasicBlock(named: "merge")

builder.buildCondBr(condition: test, then: thenBB, else: elseBB)

// MARK: Then Block
builder.positionAtEnd(of: thenBB)
// local = 1/89, the fibonacci series (sort of)
let thenVal = FloatType.double.constant(1/89)
// Branch to the merge block
builder.buildBr(mergeBB)

// MARK: Else Block
builder.positionAtEnd(of: elseBB)
// local = 1/109, the fibonacci series (sort of) backwards
let elseVal = FloatType.double.constant(1/109)
// Branch to the merge block
builder.buildBr(mergeBB)

// MARK: Merge Block
builder.positionAtEnd(of: mergeBB)
let phi = builder.buildPhi(FloatType.double, name: "phi_example")
phi.addIncoming([
  (thenVal, thenBB),
  (elseVal, elseBB),
])
builder.buildStore(phi, to: local)
let ret = builder.buildLoad(local, name: "ret")
builder.buildRet(ret)

// Setup the JIT
let jit = try! JIT(module: module, machine: TargetMachine())
typealias FnPtr = @convention(c) (Bool) -> Double
// Retrieve a handle to the function we're going to invoke
let fnAddr = jit.addressOfFunction(name: "calculateFibs")
let fn = unsafeBitCast(fnAddr, to: FnPtr.self)
// Call the function!
print(fn(true)) // 0.00917431192660551...
print(fn(false)) // 0.0112359550561798...
```

which outputs: 

```Fatal error: 'try!' expression unexpectedly raised an error: could not initialize JIT: Interpreter has not been linked in.: file /BuildRoot/Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-900.0.74.1/src/swift/stdlib/public/core/ErrorType.swift, line 181```

Could you please provide any directions on how to fix this? Thanks.

**Comment by CodaFi** [comment]
I have [already fixed this](https://github.com/llvm-swift/LLVMSwift/commit/6d834743718d7faa64f8b3f5edee5f4d8d1078f6#diff-c6ea32873c6a794467580f411ed4f64bR48).  We need to put out a release for this...  In the mean time you can pull from master.

**Comment by CodaFi** [comment]
Resolved in 0.2.2
